src/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+        integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <title>TODO List</title>
+</head>
+
+<body>
+    <div class="root">
+        <div class="title">
+            <p>Today's To Do</p><i class="fas fa-sync-alt"></i>
+        </div>
+        <div class="container-todo">
+            <input class='input-element' placeholder="Add to your list..." />
+            <ul class='ul-element'></ul>
+        </div>
+    </div>
+</body>
```

Comment:
- I suggest using semantic tags instead of general-purpose `div`s. That makes your code more meaningful for SEOs, Screen Readers, and other developers.

Reply:
Thank you for your suggestion, I going to change this tag's :+1: